x-pack/plugins/cloud_security_posture/public/pages/rules/rules_flyout.tsx

```diff
@@ -65,7 +65,7 @@ export const RuleFlyout = ({ onClose, rule, toggleRule }: RuleFlyoutProps) => {
           ))}
         </EuiTabs>
       </EuiFlyoutHeader>
-      <EuiFlyoutBody>
+      <EuiFlyoutBody key={tab}>
```

Comment:
changing `key` will force a re-mount of the component 
which is the same as wrapping each tab with its own `EuiFlyoutBody`